File: src/sdk-reference/cpp/1/collection/update-specifications/snippets/update-specifications.cpp
try {
  std::string specifications = R"({
    "strict": false,
    "fields": {
      "license": {
        "mandatory": true,
        "type": "string"
      }
    }
  })";

  std::string updated_specifications =
    kuzzle->collection->updateSpecifications("nyc-open-data", "yellow-taxi", specifications);

  std::cout << updated_specifications << std::endl;
  // {"strict":false,"fields":{"license":{"mandatory":true,"type":"string"}}}
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/update-specifications/snippets/update-specifications.cs
try {
  string specifications = @"{
    ""strict"": false,
    ""fields"": {
      ""license"": {
        ""mandatory"": true,
        ""type"": ""string""
      }
    }
  }";

  string updated_specifications =
    kuzzle.collection.updateSpecifications("nyc-open-data", "yellow-taxi", specifications);

  Console.WriteLine(updated_specifications);
  // {"strict":false,"fields":{"license":{"mandatory":true,"type":"string"}}}
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
